File: OLSource1/v1/VS_officedev/codesnippet/CSharp/accessing-data-in-documents-on-the-server_2.cs
                    serverDocument1 = new ServerDocument(documentPath);
                    CachedDataHostItem hostItem1 = 
                        serverDocument1.CachedData.HostItems["ExcelWorkbook1.Sheet1"];
                    CachedDataItem dataItem1 = hostItem1.CachedData["CachedString"];

                    if (dataItem1 != null &&
                        Type.GetType(dataItem1.DataType) == typeof(string))
                    {
                        dataItem1.SerializeDataInstance("This is the new cached string value.");
                        serverDocument1.Save();
                    }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/accessing-data-in-documents-on-the-server_2.vb
                serverDocument1 = New ServerDocument(documentPath)
                Dim hostItem1 As CachedDataHostItem = _
                    serverDocument1.CachedData.HostItems("ExcelWorkbook1.Sheet1")
                Dim dataItem1 As CachedDataItem = hostItem1.CachedData("CachedString")

                If dataItem1 IsNot Nothing AndAlso _
                    Type.GetType(dataItem1.DataType).Equals(GetType(String)) Then

                    dataItem1.SerializeDataInstance("This is the new cached string value.")
                    serverDocument1.Save()
                End If